synfig-studio/src/gui/app.cpp

```diff
@@ -3826,14 +3827,17 @@ App::dialog_paragraph(const std::string &title, const std::string &message,std::
 
 	Gtk::Label* label = manage(new Gtk::Label(message));
 	label->show();
-	dialog.get_content_area()->pack_start(*label);
+	dialog.get_content_area()->pack_start(*label, false, false);
```

Comment:
I suggest to use `pack_start(Widget& child, PackOptions options = PACK_EXPAND_WIDGET, guint padding = 0)` override.
It will be more clear what is happening here.